### PR TITLE
feat: create attribute-form component

### PR DESCRIPTION
**Pull request by luqven**
This PR creates the `AttributeForm` component that opens as a modal inside the extension. This component allows you to set/update the `alt` and `title` attributes of a selected image.

This PR also creates the `ButtonList` icon.

This PR also includes some small styling adjustments to account for the `rem` value for SFCC bot being `16px`.

## Video
[alt-title-settings.mov](https://graphite-user-uploaded-assets.s3.amazonaws.com/vkhjmXjzdqOiU0HS7RdK/50754f26-b91f-4949-8117-8ba9610c2899/alt-title-settings.mov)

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #283** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/283" target="_blank">Graphite</a> 
    * **PR #284** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/284" target="_blank">Graphite</a> 
      * **PR #285** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/285" target="_blank">Graphite</a> 
        * **PR #286** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/286" target="_blank">Graphite</a> 
          * **PR #290** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/290" target="_blank">Graphite</a> 
            * **PR #288** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/288" target="_blank">Graphite</a> 
              * **PR #287** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/287" target="_blank">Graphite</a> 
                * **PR #289** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/289" target="_blank">Graphite</a> 
                  * **PR #292** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/292" target="_blank">Graphite</a> 
                    * **PR #293** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/293" target="_blank">Graphite</a> 
                      * **PR #294** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/294" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/layouts/Modal.module.scss [hunk 1]
```diff
@@ -33,9 +33,9 @@
 }
 
 .content {
-  background-color: white;
+  // background-color: white;
```
Comment (frederickfogerty) [inline frontend/src/components/layouts/Modal.module.scss:36]: This should be removed before merging

File: frontend/src/components/forms/attributes/AttributeForm.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <label className={styles.formLabel}>
+          <MetaSvg />
+          <p>Alt text</p>
+        </label>
+        <input
+          type="text"
+          // Extension will default to this if not provided.
+          // Important to surface this here to the user to make it
+          // clear what the default is.
+          // placeholder={placeholder}
```
Comment (frederickfogerty) [inline frontend/src/components/forms/attributes/AttributeForm.tsx:43]: What does this mean here? The placeholder is currently commented out.

File: frontend/src/components/forms/attributes/AttributeForm.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          value={alt}
+          onChange={(e) => setAlt(e.target.value)}
+        />
+        <label className={styles.formLabel}>
+          <ButtonListSvg />
+          <p>Title</p>
+        </label>
+        <input
+          type="text"
+          // placeholder={placeholder}
```
Comment (frederickfogerty) [inline frontend/src/components/forms/attributes/AttributeForm.tsx:53]: This is also commented out

**Review by frederickfogerty — CHANGES_REQUESTED** [review]
See inline comments

File: frontend/src/components/forms/attributes/AttributeForm.tsx (hunk 2 above)
Comment (luqven) [reply frontend/src/components/forms/attributes/AttributeForm.tsx:43]: Yup apols. This was before I changed this into a controlled form. Will remove comments.

File: frontend/src/components/layouts/Modal.module.scss (hunk 1 above)
Comment (luqven) [reply frontend/src/components/layouts/Modal.module.scss:36]: 100% thanks goodcatch

**Review by frederickfogerty — COMMENTED** [review]
LGTM

**Comment by luqven** [comment]
@frederickfogerty assuming this was closed in error?

**Pull request merged**